.editorconfig

```diff
@@ -234,6 +234,9 @@ dotnet_diagnostic.IDE0065.severity = silent
 # IDE0072: Populate switch
 dotnet_diagnostic.IDE0072.severity = silent
 
+# IDE0090: Use 'new(...)'
```

Comment:
This is redundant- the style category entry above already bumps all IDE style diagnostics to be warning by default.